test/functional/core/startup_spec.lua

```diff
@@ -441,6 +441,7 @@ describe('user config init', function()
 
   before_each(function()
     rmdir(xhome)
+    rmdir(xconfig)
```

Comment:
Removing xconfig isn't necessary to fix the tests, but this seems like something we should do for consistency? @teto

Reply:
why? it's a nested directory.